docs/mobile-apps/automated-testing/espresso-xcuitest/espresso-capture.md

```diff
@@ -0,0 +1,39 @@
+---
+id: espresso-capture
+title: Espresso Screenshot Capture
+sidebar_label: Espresso Screenshot Capture
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The Espresso Screenshot Capture functionality allows you to easily take a snapshot of the error as to support with visual verifications of your debugging process. These screenshots also help in identifying any errors or bugs within your application.
+
+
+## Capturing Screenshots
+
+To take screenshots during an Espresso test run, Sauce Labs provides the code you can add to your Espresso tests. See our [sample app repository](https://github.com/saucelabs/my-demo-app-android/blob/1d2d85aad21bc76878d6aa4d77aefd889b59d3c9/app/src/androidTest/java/com/saucelabs/mydemoapp/android/screenshot/SauceLabsCustomScreenshot.java) and copy the utility class into your project. Make sure to handle exceptions in your test code to ensure a proper test run. This feature is supported on Android 4.3 and above.
```

Comment:
Since the sample app is linked in the steps, it could be removed from the description.
```suggestion
To take screenshots during an Espresso test run, follow the steps below. Make sure to handle exceptions in your test code to ensure a proper test run. This feature is supported on Android 4.3 and above.
```